mParticle-Radar/MPKitRadar.m

```diff
@@ -103,28 +102,77 @@ - (MPKitExecStatus *)didBecomeActive {
     return [[MPKitExecStatus alloc] initWithSDKCode:[MPKitRadar kitCode] returnCode:MPKitReturnCodeSuccess];
 }
 
-#pragma mark Events
+#pragma mark User attributes and identities
+
+- (NSString*) getMpId:(FilteredMParticleUser *)user {
+    if (user != nil && user.userId != nil && user.userId.integerValue != 0) {
```

Comment:
Is there a reason we check `!= 0` on iOS, but not on Android?

Reply:
will do the same on android